arabic/dream/dream.md

```diff
@@ -2,5 +2,4 @@
 احلم......الاحلام تتحقق اذا صدقة النية
 ولكن الاحلام لا تتحقق الا بالتوكل على الله والاخذ بالاسباب
 الان انت فى بداية الطريق ماذا ستفعل
-1- [ستبدأ فى التعلم] (https://github.com/udacity/create-your-own-adventure/blob/master/arabic/learn/learn.md
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.